src/components/chart components/chartsFunctions.js

```diff
@@ -0,0 +1,109 @@
+const dateFormatter = (date) => date.toLocaleDateString('en-US', { year: "numeric", month: "short", day: "numeric" });
```

Comment:
this file can be named chart.utils.js and could/should sit inside an utils folder, not tied to the chart components. the folder can be rewritten as "chart" without the component suffix, it is unnecessary since it is already in the components folder